_posts/2016-08-31-sandstorm-for-work-ready.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: "Sandstorm for Work is Ready"
+author: Kenton Varda
+authorUrl: https://github.com/kentonv
+imageUrl: https://sandstorm.io/news/images/sfw.png
+---
+
+Today, we're announcing that Sandstorm for Work is no longer in beta. Companies large and small -- ourselves included -- have been getting work done using Sandstorm for months. It's time for you to join us!
+
+### What is Sandstorm for Work?
+
+<img style="float: right; width: 283px; height: 221px;" src="/images/business-ldap.png">
+
+Do you wish you could use web services like Google Apps, Slack, Trello, Dropbox, and others, but can't for security, privacy, or compliance reasons? Frustrated by the setup and maintenance costs of most self-hosted solutions? Need to integrate with your corporate single-sign-on (LDAP, SAML, Active Directory) and enforce company-wide access control policies?
+
+Sandstorm is a suite of web-based productivity software which you can deploy on your own servers with minimal effort. Any user can install the apps they need with a few clicks -- like installing apps on your phone. Apps run inside secure sandboxes with single-sign-on and uniform fine-grained access control. And everything stays up-to-date automatically, so you can set it and forget it.
+
+Sandstorm for Work is Sandstorm plus the ability to integrate with your corporate single-sign-on, priority support, and other features businesses need.
+
+### Pricing
+
+During the beta period we listened to your feedback on pricing, and we've decided to make some changes:
+
+1. We've set a new list price to $10/user/month.
+2. For a limited time, we are offering an additional **50% off** if you choose annual billing.
+3. Free trials now last 60 days.
+
+<a class="linkbutton" href="https://sandstorm.io/get-feature-key">Get Sandstorm for Work &raquo;</a>
+
+### Discounted and free keys
+
+We only think you should be paying for Sandstorm if it is helping you make money. To that end:
+
+* **Non-profits** with paid employees can receive Sandstorm for Work at half price: $5/user/month.
+* **Educational institutions** supporting faculty and students pay $5/month for each faculty member and $1/month for each student.
+* **Volunteer groups** can receive Sandstorm for Work completely free.
+* **Home users** can also receive Sandstorm for Work for free -- all you crazy geeks who run home LDAP servers for your families are covered.
```

Comment:
"crazy geeks" here can be read as insulting, especially the "crazy" part.  Maybe replace with "ubergeek"? Or maybe something like: "**Home users** can also receive Sandstorm for Work for free; if you run an LDAP server for your own family, then we want your Sandstorm server to be able to take advantage of it."